os/net/netdev/netdev_ioctl.c

```diff
@@ -1063,6 +1118,11 @@ int netdev_ioctl(int sockfd, int cmd, unsigned long arg)
 	}
 
 	/* Execute the command */
+#ifdef CONFIG_NET_LWIP
+	if (cmd == SIOCLWIP) {
```

Comment:
Please follow a below rule. they check the return value of each function not to check cmd directly.

Reply:
ok